### ISSUE TITLE
why do not apply softmax before predicting class?

### ISSUE THREAD

**Issue by MaybeShewill-CV** [rishizek/tensorflow-deeplab-v3#34]
HI I wonder why don't you apply softmax on the inference logits before you decode the predicted class in file https://github.com/rishizek/tensorflow-deeplab-v3/blob/master/deeplab_model.py line 159~161. Thanks for your reply

**Comment by MaybeShewill-CV** [comment]
@rishizek Thanks I have solved this problem:)
